corehq/ex-submodules/casexml/apps/phone/restore_caching.py

```diff
@@ -24,7 +23,6 @@ def _restore_payload_path_cache_key(domain, user_id, sync_log_id, device_id):
         domain=domain,
         prefix=RESTORE_CACHE_KEY_PREFIX,
         user_id=user_id,
-        version='2.0',
```

Comment:
V3 is highly likely to occur at some point, but there are definitely no short term plans to migrate to a new sync format.